.github/actions/build-ci-images/action.yml

```diff
@@ -34,12 +34,18 @@ runs:
     - name: "Build & Push AMD64 CI images ${{ env.IMAGE_TAG }} ${{ env.PYTHON_VERSIONS }}"
       shell: bash
       run: breeze ci-image build --push --tag-as-latest --run-in-parallel --upgrade-on-failure
-    - name: "Show dependencies to be upgraded"
+    - name: "Generate source constraints"
```

Comment:
Those constraints are useful regardless to see and they are very fast to generate, so we can generate them also in non-main branch. It's helpful to see them